extension/scripts/global/globalData.js

```diff
@@ -482,6 +482,7 @@ const DEFAULT_STORAGE = {
 				armoryFilter: new DefaultSetting({ type: "boolean", defaultValue: true }),
 				armoryWorth: new DefaultSetting({ type: "boolean", defaultValue: true }),
 				upgradeRequiredRespect: new DefaultSetting({ type: "boolean", defaultValue: true }),
+				memberInfo: new DefaultSetting({ type: "boolean", defaultValue: false }),

```

Comment:
This can be true imo. It's only a single call, and only for those with API access as well.